### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by mschwamb**
A start on updating the documentation

### CODE REVIEW

File: docs/inputs.rst [hunk 1]
```diff
@@ -73,7 +73,7 @@ An example of an orbit file in cometary format, with each object ID represented
 +-------------+----------------------------------------------------------------------------------+
 
 **Header line**
-The first row in the orbit file must be a header started with ‘!!’ to denote it as the header row::
+The first row in the orbit file must be a header started with "!!" to denote it as the header row::
```
Comment (astronomerritt) [inline docs/inputs.rst:76]: This isn't true! The first row needs to start with a column headed `ObjID`, no exclamation marks. This is a holdover from a very old version of the code.

File: docs/configfiles.rst [hunk 2]
```diff
@@ -1,96 +1,55 @@
...
+Sorcha uses a configuration files to set the majority of the various parameters required for running these software packages. The configuration file for Sorcha allows turning on and off various filters for biasing the simulated small body population to what the survey should have found. An overview of the possible options for the configuration file are described below with recommendations on what you should set these config parameters to depending on your use case.
 
 .. tip::
-  We have developed  a set of utilities that are installed alongside Sorcha that can generate a config file for Objects in Field (See :ref:`makeConfigOIF`) and one for Sorcha (See :ref:`makeConfigPP`). 
+  We have developed  a set of utilities that are installed alongside Sorcha that can generate a config file for Sorcha (See :ref:`makeConfigPP`).
```
Comment (astronomerritt) [inline docs/configfiles.rst:7]: double space between "developed" and "a"

File: docs/configfiles.rst [hunk 3]
```diff
@@ -1,96 +1,55 @@
-Configuration Files
+Configuration File
 =====================
 
-Sorcha and Objects In Field both use configuration files to set the majority of the various parameters required for running these software packages. The configuration file for Sorcha allows turning on and off various filters for biasing the simulated small body population to what the survey should have found. An overview of the possible options for the config files are described below with recommendations on what you should set these config parameters to depending on your use case.
+Sorcha uses a configuration files to set the majority of the various parameters required for running these software packages. The configuration file for Sorcha allows turning on and off various filters for biasing the simulated small body population to what the survey should have found. An overview of the possible options for the configuration file are described below with recommendations on what you should set these config parameters to depending on your use case.
```
Comment (astronomerritt) [inline docs/configfiles.rst:4]: should be "Sorcha uses a configuration file"

File: docs/index.rst [hunk 4]
```diff
@@ -3,23 +3,36 @@
...
-in mind but can be adapted to be used with any survey.
 
+What is Sorcha?
+=========================================================================
 
-This documentation webpage contains an installation guide and an overview of how the survey simulator 
-works. Tutorials are provided which give examples on how to determine if an object is on a field of view, 
-if it is observable with LSST, and specific guides for near earth objects, Jupiter trojans, cometary objects etc.
+Sorcha (pronounced "surk-ha") is a python Solar System survey simulator. Sorcha means light 
+or brightness in Irish and Scots gaelic, and our software is estimating the brightness of
```
Comment (astronomerritt) [inline docs/index.rst:17]: Should be "Scots **Gaelic**".

File: docs/index.rst [hunk 5]
```diff
@@ -3,23 +3,36 @@
...
+simulated Solar System small bodies and determines which ones the survey could detect in
+each of the survey's observations or based on user set critiera. Sorcha has been designed 
+with the `Vera C. Rubin Observatory Legacy Survey of Space and Time (LSST) <https://www.lsst.org>`_ 
+in mind. The software has a module design, and  with some effort it can be adapted to be
```
Comment (astronomerritt) [inline docs/index.rst:21]: "The software has a **modular** design"

File: docs/index.rst [hunk 6]
```diff
@@ -3,23 +3,36 @@
...
+each of the survey's observations or based on user set critiera. Sorcha has been designed 
+with the `Vera C. Rubin Observatory Legacy Survey of Space and Time (LSST) <https://www.lsst.org>`_ 
+in mind. The software has a module design, and  with some effort it can be adapted to be 
+used with any survey.   
 
 .. warning::
    This documentation site and the software packages it describes are under
    active development. Validation is still on-going. DO NOT USE this for science
-   purposes just yet.
+   purposes just yet. WE REALLY MEANS THIS. THE CODEBASE IS UNDER HEAVY DEVLEOPMENT.
```
Comment (astronomerritt) [inline docs/index.rst:27]: "WE REALLY **MEAN** THIS. THE CODE IS UNDER HEAVY **DEVELOPMENT.**" Although the typos do add to the sense of urgency!

File: docs/index.rst [hunk 7]
```diff
@@ -3,23 +3,36 @@
...
  
+
+Welcome to Sorcha's documentation!
+=========================================================================
+
+This documentation site contains an installation guide, an overview of how Sorcha 
+works, tutorials, and demostration notebooks that show how each of the various filters works.
```
Comment (astronomerritt) [inline docs/index.rst:34]: "and **demonstration** notebooks"

File: docs/index.rst [hunk 8]
```diff
@@ -3,23 +3,36 @@
...
  
+
+Welcome to Sorcha's documentation!
+=========================================================================
+
+This documentation site contains an installation guide, an overview of how Sorcha 
+works, tutorials, and demostration notebooks that show how each of the various filters works. 
+
 .. seealso::
    We have a summary paper (currently in prep) that provides and a broad summary
```
Comment (astronomerritt) [inline docs/index.rst:37]: "that **provides a broad summary**"

File: docs/inputs.rst [hunk 9]
```diff
@@ -26,7 +26,7 @@ Orbit File
 -----------------
 
 .. note::
-  The orbit file is used by  **Objects in Field** and **Sorcha**.
+  The orbit file is used by **Sorcha**.
 
 This is a file which contains the orbital information of a set of synthetic objects. The orbital parameters must be **heliolcentric**, and orbits can be define in **Cometary(COM)  or Keplerian (KEP)** formats. Each simulated planetesimals within the synthetic population must be be given it's own unique object ID (ObjID). The file can be **white space separated**  or **comma value separated (CSV)** format. The first line of the orbit file is a header line starting with !! that specifies what each of the columns are.
```
Comment (astronomerritt) [inline docs/inputs.rst:31]: "each simulated **planetesimal**" and "must be given **its** own unique object ID"

Also, the first line of the orbit file should NOT start with !!.

File: docs/inputs.rst [hunk 10]
```diff
@@ -159,13 +159,13 @@ Rubin Observatory will survey the sky in six broadband (optical filters), *u, g,
...
 
 .. note::
-  *  In the config file you can decide which filters you want have Sorcha run on and specify which filter is the main filter that the absolute magnitude is defined for. You only need to provide colors for those fliters specified in the config file. 
+  In the config file you can decide which filters you want have Sorcha run on and specify which filter is the main filter that the absolute magnitude is defined for. You only need to provide colors for those fliters specified in the config file.
```
Comment (astronomerritt) [inline docs/inputs.rst:162]: "which filters you want **to** have Sorcha run"

File: docs/troubleshooting.rst [hunk 11]
```diff
@@ -1,16 +1,24 @@
...
+
 Troubleshooting
 =================
 
 
 Using Relative File Paths
 ---------------------------------------------------------------
 
-OIF assumes explicit file paths so ../this_directory will cause errors. It is best to give the full output and input paths for directories and input file/ouput file locations.
+We recommend using explicit file paths so ../this_directory will cause errors. It is best to give the full
```
Comment (astronomerritt) [inline docs/troubleshooting.rst:11]: I would word this as "We recommend using explicit file paths **as ../this_directory may** cause errors". It shouldn't as long as the user is sensible and runs the command in the same directory they've defined the paths relative to, but of course users are often not sensible (and I include myself in that).

File: docs/troubleshooting.rst [hunk 12]
```diff
@@ -23,17 +31,16 @@ Then it is likely that you are using the older or newer version of the (simulate
...
 Mismatch in Inputs 
 ---------------------
-There are several files which are passed into Sorcha. These are the 
-orbit file, the physical parameter file and an optional cometary parameter file. Each provide
-specific information about the synthetic population that is being analysed.
-
-Within these files, it is necessary to specify an entry for every object. The Sorcha 
-code will run a check to ensure that all entries have an associated 
-orbit and physical/cometary parameter value, so if you get an error like::
+There are several files associated with the synthetic small bodies  which are passed into Sorcha. These are
```
Comment (astronomerritt) [inline docs/troubleshooting.rst:34]: double space between "bodies" and "which"

File: docs/troubleshooting.rst [hunk 13]
```diff
@@ -23,17 +31,16 @@ Then it is likely that you are using the older or newer version of the (simulate
...
+the orbit file, the physical parameter file and an optional complexy parameters file and optional ephemeris
```
Comment (astronomerritt) [inline docs/troubleshooting.rst:35]: "physical **parameters** file" and also "and an optional **complex** parameters file"

File: docs/troubleshooting.rst [hunk 14]
```diff
@@ -23,17 +31,16 @@ Then it is likely that you are using the older or newer version of the (simulate
...
-There are several files which are passed into Sorcha. These are the 
-orbit file, the physical parameter file and an optional cometary parameter file. Each provide
-specific information about the synthetic population that is being analysed.
-
-Within these files, it is necessary to specify an entry for every object. The Sorcha 
-code will run a check to ensure that all entries have an associated 
-orbit and physical/cometary parameter value, so if you get an error like::
+There are several files associated with the synthetic small bodies  which are passed into Sorcha. These are
+the orbit file, the physical parameter file and an optional complexy parameters file and optional ephemeris 
+file (if not using the ephemeris generator within sorcha. Each provide specific information about the
```
Comment (astronomerritt) [inline docs/troubleshooting.rst:36]: missing an end bracket here!

File: docs/troubleshooting.rst [hunk 15]
```diff
@@ -23,17 +31,16 @@ Then it is likely that you are using the older or newer version of the (simulate
...
-
-Within these files, it is necessary to specify an entry for every object. The Sorcha 
-code will run a check to ensure that all entries have an associated 
-orbit and physical/cometary parameter value, so if you get an error like::
+There are several files associated with the synthetic small bodies  which are passed into Sorcha. These are
+the orbit file, the physical parameter file and an optional complexy parameters file and optional ephemeris 
+file (if not using the ephemeris generator within sorcha. Each provide specific information about the 
+synthetic population that is being analysed. Within these files, it is necessary to specify an entry for every 
+object. The Sorcha code will run a check to ensure that all entries have an associated orbit and 
+physical/complex physical  parameter value, so if you get an error like::
```
Comment (astronomerritt) [inline docs/troubleshooting.rst:39]: double space between "physical" and "parameter"

File: docs/troubleshooting.rst [hunk 16]
```diff
@@ -23,17 +31,16 @@ Then it is likely that you are using the older or newer version of the (simulate
...
 
    ERROR: PPCheckOrbitAndColourMatching: input colour/cometary parameter and orbit files do not match.
 
-then make sure that for each file (orbit, ephemerides and physical/cometary parameters) contains information 
+then make sure that for each file (orbit, ephemerides and phs) contains information
```
Comment (astronomerritt) [inline docs/troubleshooting.rst:43]: Is using "phs" deliberate here or should that read "physical parameters"?

File: docs/uninstall.rst [hunk 17]
```diff
@@ -1,16 +1,13 @@
 Uninstalling 
 =================
 
-Uninstalling OIF
-~~~~~~~~~~~~~~~~~~~
-To uninstall::
+If you have installed sorcha through the pip with the package on pypy. Then you can uninstall the package the usual way with pip::
```
Comment (astronomerritt) [inline docs/uninstall.rst:4]: "installed sorcha through pip", delete **the**

File: docs/whatsspdoesnotdo.rst [hunk 18]
```diff
@@ -1,14 +1,22 @@
...
-- Cometary outbursts
-- YORP (Yarkovsky–O'Keefe–Radzievskii–Paddack) effect
-- Non-gravitational forces
-- Collisions and breakup events
+- Non-gravitational forces including cometary outgassing or YORP (Yarkovsky–O'Keefe–Radzievskii–Paddack) effect. Although not directly handled,you can input your own ephemeris files that account for these effects if required.  
+- Removing simulated objects due to small body collisions and breakup events
+
+
+.. seealso::
+   We do have methods for  users to easily develop their own functions for adjusting the apparent
```
Comment (astronomerritt) [inline docs/whatsspdoesnotdo.rst:18]: Double space between "for" and "users".

File: docs/whatsspdoesnotdo.rst [hunk 19]
```diff
@@ -1,14 +1,22 @@
...
-- Collisions and breakup events
+- Non-gravitational forces including cometary outgassing or YORP (Yarkovsky–O'Keefe–Radzievskii–Paddack) effect. Although not directly handled,you can input your own ephemeris files that account for these effects if required.  
+- Removing simulated objects due to small body collisions and breakup events
+
+
+.. seealso::
+   We do have methods for  users to easily develop their own functions for adjusting the apparent 
+   magnitude of the simulated objects  due to cometary activity, rotational light curves, cometary 
+   outbursts, etc. We have some basic functionality already built for simple sinusoidal rotational 
+   light curves and cometary activity. Further details can be found  in the
```
Comment (astronomerritt) [inline docs/whatsspdoesnotdo.rst:21]: Double space between "found" and "in".

**Review by astronomerritt — APPROVED** [review]
Most of my comments here are about typos. Feel free to ignore those if you'd rather do a proof-read run later. There are a couple of very minor requests for clarification.

HOWEVER: there are a couple places which state that the header of the orbits file has to begin with !!. This is no longer true: it should start with the "ObjID" column heading, no exclamation marks.

That's the only really important change and you can obviously address it in a later PR, I'm approving this for merge. Other than that this looks great so far!

File: docs/inputs.rst (hunk 1 above)
Comment (mschwamb) [reply docs/inputs.rst:76]: @steph. This is not complete. Let’s not go through every change. Required. It’s definitely not complete.

File: docs/index.rst (hunk 6 above)
Comment (mschwamb) [reply docs/index.rst:27]: I’ll deal with typos later

File: docs/inputs.rst (hunk 1 above)
Comment (astronomerritt) [reply docs/inputs.rst:76]: That's fine - if you check my overall review, I've approved the PR and noted that all the things I've commented on can easily be addressed much later.

File: docs/inputs.rst [hunk 20]
```diff
@@ -159,13 +159,13 @@ Rubin Observatory will survey the sky in six broadband (optical filters), *u, g,
...
 
 .. note::
-  *  In the config file you can decide which filters you want have Sorcha run on and specify which filter is the main filter that the absolute magnitude is defined for. You only need to provide colors for those fliters specified in the config file. 
+  In the config file you can decide which filters you want have Sorcha run on and specify which filter is the main filter that the absolute magnitude is defined for. You only need to provide colors for those fliters specified in the config file. 
 
 .. warning::
-  * You must use the same phase curve prescription for all simulated objects. If you want to use different phase curve prescriptions for different synthetic populations, you will need to run them in separate input files to Sorcha
+  You must use the same phase curve prescription for all simulated objects. If you want to use different phase curve prescriptions for different synthetic populations, you will need to run them in separate input files to Sorcha
```
Comment (jeremykubica) [inline docs/inputs.rst:165]: Not a comment on the text itself, but rather a general question: Do we want to change this in the future and provide the ability to specify per-object phases curves?

**Pull request merged**